packages/svelte/types/index.d.ts

```diff
@@ -934,7 +934,8 @@ declare module 'svelte/compiler' {
 			| 'each'
 			| 'snippet'
 			| 'store_sub'
-			| 'legacy_reactive';
+			| 'legacy_reactive'
+			| 'template';
```

Comment:
The JSDoc above the union type has a bit of explanation for each value - could you add a bit there?

Side note: I'm a bit surprised no other adjustments were necessary, `derived` was chosen as a value to get the `$.get` transformation (maybe that's because of the transformers PR a while back which makes it not as reliant on that anymore).